flutter/beagle/lib/default/default_storage.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:beagle/interface/storage.dart';
+import 'package:flutter/cupertino.dart';
```

Comment:
Since you are not using any specific Cupertino component, how about importing directly from `widgets.dart`?
```suggestion
import 'package:flutter/widgets.dart';
```

Reply:
You're right. But I used this import just use debugPrint. It shouldn't have been comitted. I removed both the import and the unnecessary log.